### PR TITLE
dracut, initcpio: make libgcc_s search universal

### LINKED ISSUE #427: libgcc_s.so is sometimes missing in mkinitcpio-based images
This will take a bit of care / research to resolve.

### PR DESCRIPTION
**Pull request by ahesford**
I'm still working on this and haven't tested anything, but this should close #427.

### CODE REVIEW

**Comment by ahesford** [comment]
Initial tests show that `libgcc_s.so` and `libgcc_s.so.1` make it into a `dracut` testbed image generated on a Void host, although I make no provision for `libgcc_s.so` so I don't know where it's coming from. In a Void-created `mkinitcpio` image, `libgcc_s.so.1` appears in the image and `mbuffer` runs without complaint (although I didn't actually pipe data through it).

File: dracut/module-setup.sh [hunk 1]
```diff
@@ -68,25 +68,19 @@ install() {
...
-    if command -v gcc-config >/dev/null 2>&1; then
-      dracut_install "/usr/lib/gcc/$(s=$(gcc-config -c); echo "${s%-*}/${s##*-}")/libgcc_s.so.1"
-      _ret=$?
-    # Otherwise, use dracut's library installation function to find the right one
-    elif ! inst_libdir_file "libgcc_s.so*"; then
-      # If all else fails, just try looking for some gcc arch directory
-      dracut_install /usr/lib/gcc/*/*/libgcc_s.so*
-      _ret=$?
+  # shellcheck disable=SC2154
+  if [ -z "${zfsbootmenu_skip_gcc_s}" ]; then
```
Comment (zdykstra) [inline dracut/module-setup.sh:72]: This should probably be documented in zfsbootmenu.7 for both the dracut / initcpio options sections.

Comment (zdykstra) [reply dracut/module-setup.sh:72]: I'm okay with leaving this undocumented until it becomes a real issue.

Comment (ahesford) [reply dracut/module-setup.sh:72]: I decided to document the behavior. Let me know if the wording needs adjustment.

**Review by zdykstra — APPROVED** [review]
That seems very clear to me - and is niche enough that I expect few people will ever come into contact with it.

**Pull request merged**